calliope/backend/pyomo/constraints/capacity.py

```diff
@@ -130,8 +132,13 @@ def storage_capacity_constraint_rule(backend_model, loc_tech):
             \\quad \\forall loc::tech \\in loc::techs_{store}
 
     """
-
-    return get_capacity_constraint(backend_model, 'storage_cap', loc_tech)
+    ratio = get_param(backend_model, 'storage_cap_ratio', loc_tech)
+    if po.value(ratio) is not False and po.value(ratio) is not None:
+        return backend_model.storage_cap[loc_tech] == (
+            backend_model.energy_cap[loc_tech] * ratio
```

Comment:
All the literature defines it the other way, i.e. power to energy ratio, not energy to power ratio. No matter the name given to the parameter, it should probably follow that convention